src/TLuaInterpreter.cpp

```diff
@@ -11296,25 +11341,33 @@ int TLuaInterpreter::registerAnonymousEventHandler(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#expandAlias
 int TLuaInterpreter::expandAlias(lua_State* L)
 {
-    string luaSendText;
+    QString payload;
     if (!lua_isstring(L, 1)) {
-        lua_pushstring(L, "expandAlias: wrong argument type");
-        lua_error(L);
-        return 1;
+        lua_pushfstring(L, "expandAlias: bad argument #1 type (text to parse as string expected, got %s!)", luaL_typename(L, 1));
```

Comment:
I know, but for simplicity's sake, and because people only really use expandAlias for aliases, we should call it an alias so it's easier to understand.